app/presenters/step_by_step_page_presenter.rb

```diff
@@ -57,6 +57,18 @@ def summary_list_params
     params
   end
 
+  def broken_link_status_text(step)
+    if step.broken_links?
+      "#{step.broken_links.count} broken #{'link'.pluralize(step.broken_links.count)} found"
+    elsif step.link_report?
+      "No broken links"
+    end
+  end
+
+  def broken_link_status_classes(step)
+    "govuk-tag govuk-tag--#{step.broken_links? ? 'warning' : 'success'}"
```

Comment:
I'm not convinced of the value of moving all of the css code into the presenter. It feels like this should live in the view as it feels like it's less clear what options are being presented to users.

Presenters are usually just used to [manipulate data](https://stackoverflow.com/questions/36458688/what-is-the-rails-presenters-folder-for) for presentation in the view.